src/ChainSafe.Gaming.Unity.EthereumWindow/EthereumWindowController.cs

```diff
@@ -0,0 +1,159 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Evm;
+using ChainSafe.Gaming.Unity.EthereumWindow.Dto;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Hex.HexTypes;
+using Nethereum.JsonRpc.Client.RpcMessages;
+using Newtonsoft.Json;
+using UnityEngine;
+
+namespace ChainSafe.Gaming.Unity.EthereumWindow
+{
+    /// <summary>
+    /// A controller script for making awaitable JsonRPC requests to browser wallets via window.ethereum.
+    /// Make sure EthereumWindow.jslib is present in your Web3.Unity Package.
```

Comment:
can't we ourselves include it ? :D why would the end user need to check it's presence?

Reply:
We already have it in the package but just incase Users are just using our solution and not the package, we ought to mention this